addons/mail/push-to-talk-extension/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "manifest_version": 3,
+    "name": "Discuss Push-to-talk",
+    "description": "Empowers Odoo Discuss with system-wide Push-to-Talk",
+    "version": "0.0.0.1",
```

Comment:
Before we merge it, I'd suggest you change the version to something that sounds less like the extension is still in development

Reply:
I think it's best practice:
![image](https://github.com/odoo/odoo/assets/48757558/e26e22ba-a5b9-4cf9-8c29-d1e87ef2deb7)
https://developer.chrome.com/docs/webstore/prepare/